modin/engines/base/io/text/csv_reader.py

```diff
@@ -97,6 +96,7 @@ def _read(cls, filepath_or_buffer, **kwargs):
         quotechar = kwargs.get("quotechar", '"').encode(
             encoding if encoding is not None else "UTF-8"
         )
+        is_quoting = kwargs.get("quoting", "") != csv.QUOTE_NONE
```

Comment:
it was processed, but it always was calls like these
```python
kwargs.get("quoting", "") != csv.QUOTE_NONE
```
since we're using it multiple times I've saved the result into a variable